extensions/security/runtime/src/main/java/io/quarkus/security/runtime/QuarkusSecurityIdentity.java

```diff
@@ -205,6 +208,48 @@ public Builder addAttributes(Map<String, Object> attributes) {
             return this;
         }
 
+        /**
+         * Adds a string permission.
+         *
+         * @param permission The permission in a String format
+         * @return This builder
+         */
+        public Builder addPermission(String permission) {
+            return addStringPermissions(Set.of(permission));
+        }
+
+        /**
+         * Adds permissions as strings.
+         *
+         * @param permissions The permissions in a String format
+         * @return This builder
+         */
+        public Builder addStringPermissions(Set<String> permissions) {
```

Comment:
I understand why you add to do that here but it's weird to have the singular and plural versions different.

Maybe `addPermissionAsString()` / `addPermissionsAsString()` would be better?